kibbeh/src/icons/WinMinimizeIcon.tsx

```diff
@@ -5,15 +5,15 @@ function WinMinimizeIcon(props: React.SVGProps<SVGSVGElement>) {
     <svg
       xmlns="http://www.w3.org/2000/svg"
       width="15"
-      height="1.24"
-      viewBox="0 0 15 1.24"
+      height="1"
```

Comment:
don't change the SVG's height bro, that flattens it out

Reply:
oh

Reply:
but isn't that the only way to fix it if you remove the height customization on the <WinMinimizeIcon /> in ElectronHeader.tsx?

Reply:
flattening the svg isn't the way to go, better way would be finding a higher quality svg

Reply:
Did you just export them out of figma or what?

Reply:
no @ajmnz sent me...

Reply:
then it's probably from figma

Reply:
well if i change all 1.24's to 1 it would need to flatten it

Reply:
revert this, we will try finding a better svg if its a problem